app/models/rdv.rb

```diff
@@ -62,6 +63,7 @@ def nested_lieu_attributes
   validate :lieu_is_not_disabled_if_needed
   validate :starts_at_is_plausible
   validate :duration_is_plausible
+  validates :name, length: { maximum: 50 }
```

Comment:
Cette nouvelle validation a créé un bug un peu embêtant : c'est maintenant impossible de s'inscrire à des rdv qui ont un intitulé de plus de 50 caractères : https://sentry.incubateur.net/organizations/betagouv/issues/17534/?environment=production&query=is%3Aunresolved
On aurait pu le vérifier en amont avec un `Rdv.where("length(name) > 50").count` (660 rdvs).
Est-ce qu'on peut plutôt tronquer la fin de l'intitulé dans le sms plutôt que d'ajouter une contrainte au niveau du modèle pour quelque chose d'unique aux sms ?